src/type.ts

```diff
@@ -0,0 +1,34 @@
+export type employee = {
+  dateOfBirth: string;
+  dateOfJoin: string;
+  department: number;
+  email: string;
+  fullName: string;
+  id: number;
+  role: number;
+  skills: number[];
+  workLocation: string;
+  imageSrc: string;
+};
+
+export type department = {
+  departmentID: number;
+  departmentName: string;
+};
+
+export type skill = {
+  skillID: number;
+  skill: string;
+};
+
+export type role = {
+  roleID: number;
+  role: string;
+};
+
+export type fullData = {
```

Comment:
fullData can be renamed as CompanyData or even as FirebaseData. "fulldata" is confusing